src/Models/IoTProject.ts

```diff
@@ -143,6 +143,10 @@ export class IoTProject {
         this.componentList.push(functionApp);
       }
 
+      this.componentList.forEach(item => {
+        item.checkPrerequisites();

```

Comment:
checkPrerequisites is async function

Reply:
The order is not necessary here, so `await` is not required.